_includes/markdown/Systems.md

```diff
@@ -252,3 +253,94 @@ Once tuning moves beyond these main items, performance improvements will be mino
+
+### Memcached
+
+Memcached is simpler than Redis and has fewer features. While Redis can be used as a full database, memcached is only a key-value store. This simplicity is by design and makes Memcached a very low maintenance tool. 
+
+#### High Availability
+
+Memcached can be installed on multiple servers that can combine into a memcached “pool”.  Keeping with the theme of simplicity, each memcached server in the pool knows nothing about the other instances in the pool and operates completely independently.  It is up to the client (in this case, WordPress and PHP) to distribute data across the pool of memcached servers however it sees fit.  Memcached performs no replication, failover, or connection balancing itself.  To distribute data evenly across multiple servers, the PHP extensions for memcached use a hashing strategy.  In this way, multiple PHP web servers can read and write to multiple memcached servers and all know exactly where each key is stored.  If one server fails, keys are distributed across the remaining memcached instances.  This can have a hit on performance (especially compared to replication scheme) as keys need to be regenerated to replace the missing data, but is by far the easiest type of distributed cache to setup.
+
+#### Item Size
+
+By default, memcached accepts items (key + value) of 1 MB or less. In most scenarios, this is fine and plenty of space. However, on larger WordPress sites or sites with a number of plugins, the “alloptions” array combining all autoloaded rows from the wp_options table can exceed 1 MB.  This array will be stored in memcached if memcached is in use for the object cache, unless it exceeds the memcached item size limit.  When this array is larger than the memcached item size, it can cause all sorts of odd issues and inconsistent performance.  
+
+Starting in memcached version 1.4.2, the item size is configurable via the `-I` option.  Setting the item size higher than 1 MB is not recommended unless necessary as memcached becomes less efficient as the item size increases, meaning it will take more memory to store the same amount of data.  If memory is available to accommodate a larger cache size, it is worth considering raising the item size to prevent the alloptions array from ever becoming a problem.  
+
+#### Connecting WordPress to Memcached
+
+WordPress connects to memcached through an object-cache.php drop-in plugin file placed in the wp-content folder.  The object-cache.php file will leverage a php extension to handle communicating with memcached.  There are two PHP extensions commonly used, confusingly named [php-memcached](https://pecl.php.net/package/memcache) and [php-memcached](https://pecl.php.net/package/memcached).  It is important to match the object-cache.php file with the right PHP extension, and, while there’s many object cache files that can work, 10up mostly uses one of the following:
```

Comment:
in `confusingly named [php-memcached](https://pecl.php.net/package/memcache) and [php-memcached](https://pecl.php.net/package/memcached).` The first one is php-memcache instead of php-memcached.

Reply:
I can't tell you how many times I looked at it to make sure I typed it right and I still screwed it up.  Change coming in a moment.